templates/localgov-alert-banner.html.twig

```diff
@@ -45,11 +45,9 @@
           </h2>
         {% endif %}
         
-        {% if content.short_description.0['#text'] %}
-          <div class="localgov-alert-banner__body">
-            {{ content.short_description }}
-          </div>
-        {% endif %}
+        <div class="localgov-alert-banner__body">
+          {{ content|without('title', 'link') }}
+        </div>
 
         {% if has_link %}
           <div class="localgov-alert-banner--content-link">{{ content.link }}</div>
```

Comment:
This class should be `.localgov-alert-banner__content-link` as an "element" of `.localgov-alert-banner` not a "modifier" of it.

Reply:
Shall I modify it here or as a new issue?
I assume there is some stylesheet in theme targeting this which is why it was left as is.

Reply:
Maybe leave it so, as it will not be available in the new theme that will launch soon.